cocos/2d/renderer/draw-batch.ts

```diff
@@ -45,7 +45,7 @@ export class DrawBatch2D {
     public set inputAssembler (ia: InputAssembler | null) {
         this._inputAssembler = ia;
         if (JSB) {
-            this._nativeObj.inputAssembler = ia;
+            //this._nativeObj.inputAssembler = ia;
```

Comment:
No need ? Could `if (JSB) {xxx}` be removed?

Reply:
done. removed all abundant code in draw-batch.ts